src/App.tsx

```diff
@@ -252,7 +247,7 @@ export default function App() {
               )}
             </button>
           ))}
-        </div>
+        </SimpleBar>
```

Comment:
It's almost like it was made for this project! :)

Reply:
Lol, maybe there's a SimpleInput component out there too